src/components/BackLink.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+import { ButtonLinkProps } from "@utrecht/component-library-react/dist/ButtonLink";
+import clsx from "clsx";
+import { ForwardedRef, forwardRef, PropsWithChildren } from "react";
+import { ButtonGroup } from "./ButtonGroup";
+import { ButtonLink } from "./index";
+
+type BackLinkProps = ButtonLinkProps;
+
+export const BackLink = forwardRef(
+  ({ children, className, ...restProps }: PropsWithChildren<BackLinkProps>, ref: ForwardedRef<HTMLAnchorElement>) => (
+    <ButtonGroup className={clsx("utrecht-button-group--back-link", className)}>
```

Comment:
I would make this "utrecht-back-link" and create styling for this new component instead :)

Reply:
Sounds like a great idea to me, internally it would still be using the usual link component, so the back-link is just a grouping/wrapper component like the original `ButtonGroup` was.